inc/class-cachify.php

```diff
@@ -769,6 +825,16 @@ public static function process_flush_request( $data ) {
 				);
 			}
 		}
+
+		/* Reschedule HDD Cache Cron */
+		if ( self::METHOD_HDD == self::$options['use_apc'] ) {
```

Comment:
same, === instead ==

Reply:
Is included in "Make requested changes to checks"